### PR TITLE
UI update client form

### PR DESCRIPTION
**Pull request by palant-dev**
Finished editing the file and added also an extension to ShapeStyle for fast coding with our palette of colours just use the .mainTeal and .mainBlack to call them in the context of the Color struct.
iScreen Shoter - Xcode - 230118161858

### CODE REVIEW

File: SlimApp/ClientList/ClientCard.swift [hunk 1]
```diff
@@ -8,26 +8,56 @@
...
             Spacer()
-            Divider()
-            // MARK: Test image, would be changed on client avatar
-            Image("4aces")
-                .resizable()
-                .scaledToFit()
+            ZStack {
+                Rectangle()
+                    .foregroundColor(Color.mainBlack)
+                    .frame(width: 80, height: 200, alignment: .trailing)
```
Comment (giiks131) [inline SlimApp/ClientList/ClientCard.swift]: height and alignment are not really needed for this .frame

File: SlimApp/ClientList/ClientCard.swift [hunk 2]
```diff
@@ -8,26 +8,56 @@
...
+                    }
+                    Text("16:00")
+                        .font(.title3.bold())
+                }
+            }
         }
-        .padding()
-        .frame(maxHeight: 110)
-        .background(.purple)
+        .frame(width: 370, height: 100, alignment: .leading)
```
Comment (giiks131) [inline SlimApp/ClientList/ClientCard.swift]: is alignment needed here?

**Review by giiks131 — CHANGES_REQUESTED** [review]

File: SlimApp/ClientList/ClientCard.swift (hunk 1 above)
Comment (palant-dev) [reply SlimApp/ClientList/ClientCard.swift]: Removed!

File: SlimApp/ClientList/ClientCard.swift (hunk 2 above)
Comment (palant-dev) [reply SlimApp/ClientList/ClientCard.swift]: Removed!

**Review by giiks131 — APPROVED** [review]

**Pull request merged**